content/number-theory/Eratosthenes.h

```diff
@@ -3,20 +3,29 @@
  * Date: 2009-08-26
  * License: CC0
  * Source: http://en.wikipedia.org/wiki/Sieve_of_Eratosthenes
- * Description: Prime sieve for generating all primes up to a certain limit. isprime$[i]$ is true iff $i$ is a prime.
- * Time: lim=100'000'000 $\approx$ 0.8 s. Runs 30\% faster if only odd indices are stored.
- * Status: Tested
+ * Description: \texttt{s[i]=0} if $i$ is 0, 1, else smallest
+ * prime factor of $i$. \texttt{sieve} returns sorted primes
```

Comment:
```suggestion
 * Description: s[i] = smallest prime factor of i (except for i = 0, 1).
 * \texttt{sieve} returns sorted primes.
```
Así concuerda con el estilo de por ejemplo Zfunc

También, no se si el nombre s me gusta, porque muchas veces se usa para otra cosa, quizás llamarlo pr? (no estoy muy seguro de que nombre conviene)